README.md

```diff
@@ -1,9 +1,36 @@
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1AzP33O8rnMW__7ocWJhVBXjKziJXPtim?usp=sharing) 
-[![slack](https://img.shields.io/badge/Slack-CAMEL--AI-green)](https://join.slack.com/t/camel-kwr1314/shared_invite/zt-1vy8u9lbo-ZQmhIAyWSEfSwLCl2r2eKA)
-[![Discord](https://img.shields.io/badge/Discord-CAMEL--AI-green)](https://discord.gg/CNcNpquyDc)
-<a href="https://huggingface.co/camel-ai"><img src="https://huggingface.co/datasets/huggingface/brand-assets/resolve/main/hf-logo-with-title.png" alt="Hugging Face" width="100"/></a> 
+<div style="left">
+  <a href="https://colab.research.google.com/drive/1AzP33O8rnMW__7ocWJhVBXjKziJXPtim?usp=sharing" target="_blank">
+    <img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg" />
+  </a>
+  <a href="https://join.slack.com/t/camel-kwr1314/shared_invite/zt-1vy8u9lbo-ZQmhIAyWSEfSwLCl2r2eKA" target="_blank">
+      <img alt="Slack" src="https://img.shields.io/badge/Slack-CAMEL--AI-blueviolet?logo=slack" />
+  </a>
+  <a href="https://discord.gg/CNcNpquyDc" target="_blank">
+    <img alt="Discord" src="https://img.shields.io/badge/Discord-CAMEL--AI-7289da?logo=discord&logoColor=white&color=7289da" />
+  </a>
+  <a href="https://huggingface.co/camel-ai" target="_blank">
+    <img alt="Hugging Face" src="https://img.shields.io/badge/%F0%9F%A4%97%20Hugging%20Face-CAMEL--AI-ffc107?color=ffc107&logoColor=white" />
+  </a>
+  <a href="https://twitter.com/CamelAIOrg" target="_blank">
+    <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/CamelAIOrg?style=social&color=brightgreen&logo=twitter" />
+  </a>
+</div>
+
 # CAMEL: Communicative Agents for “Mind” Exploration of Large Scale Language Model Society
 
+<div align="center">
+
+  <a>![Python 3.9+](https://img.shields.io/badge/Python-3.9%2B-brightgreen.svg)</a>
+  <a href="https://github.com/camel-ai/camel/tree/HEAD/test">![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/camel-ai/camel/pytest_apps.yml?label=tests&logo=github)</a>
```

Comment:
```suggestion
  <a href="https://github.com/camel-ai/camel/actions/workflows/pytest_package.yml">![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/camel-ai/camel/pytest_package.yml?label=tests&logo=github)</a>
```
Maybe direct to some workflow page? Though I am not sure which workflow should may need to direct to.